site/src/pages/AIBridgePage/RequestLogsPage/RequestLogsRow/RequestLogsRow.tsx

```diff
@@ -21,6 +25,47 @@ type RequestLogsRowProps = {
 	interception: AIBridgeInterception;
 };
 
+const customisedDateLocale: Intl.DateTimeFormatOptions = {
+	// Hide the year from the date
+	year: undefined,
```

Comment:
why? we'll definitely end up with logs from a year ago eventually, this feels like a bad idea